src/test/java/io/neonbee/internal/json/ConfigurableJsonFactoryTest.java

```diff
@@ -66,65 +50,6 @@ void resetToDefault() {
         ConfigurableJsonFactory.CODEC.setMaxStringLength(DEFAULT_MAX_STRING_LEN);
     }
 
-    @Test
-    @DisplayName("Databind module registration should be ignored if no Databind Codec is used")
-    void testIgnoreModuleRegistrationWithoutDatabindCodec() throws Throwable {
```

Comment:
Why are these tests no longer necessary?

Reply:
In regards of the two tests I removed, these where the tests, that wrote into final private variables inside of other classes, which is no longer possible in Java 21. I looked into the tests, essentialyl they just tested a single if (so if true was the method called or not) and because it is testing this if in regards to a static variable of a Vert.x class, there was no other way to test it, instead of overriding that static final field... so I said, becuase of only testing a single if and being the only test that requires overriding finals, it is not worth the effort to keep it.